_data/finance.yml

```diff
@@ -375,6 +375,13 @@ websites:
       - totp
     doc: https://www.taxact.com/support/24138/2019/how-is-a-verification-code-used?hideLayout=True
 
+  - name: Techniker Krankenkasse
+    url: https://www.tk.de/
+    img: techniker.png
+    tfa:
+      - proprietary
+    doc: https://www.tk.de/techniker/unternehmensseiten/unternehmen/die-tk-app/tk-app-sicherheitskonzept-2023678
+
```

Comment:
i am not sure, if "finance" is the correct category, since "Krankenkasse" means health care...